packages/x-components/src/components/display-emitter.vue

```diff
@@ -1,55 +1,58 @@
-<template>
-  <NoElement ref="root">
-    <slot />
-  </NoElement>
-</template>
-
 <script lang="ts">
-  import { defineComponent, onUnmounted, PropType, Ref, ref } from 'vue';
+  import {
+    defineComponent,
+    getCurrentInstance,
+    h,
+    onMounted,
+    onUnmounted,
+    PropType,
+    WatchStopHandle
+  } from 'vue';
   import { TaggingRequest } from '@empathyco/x-types';
   import { useEmitDisplayEvent } from '../composables/use-on-display';
   import { WireMetadata } from '../wiring';
-  import { NoElement } from './no-element';
+  import { taggingXModule } from '../x-modules/tagging/x-module';
 
-  /**
-   * A component that emits a display event when it first appears in the viewport.
-   */
+  /** A component that emits a display event when it first appears in the viewport. */
   export default defineComponent({
-    components: {
-      NoElement
-    },
+    name: 'DisplayEmitter',
+    xModule: taggingXModule.name,
```

Comment:
Should this component be relocated to the `tagging` x-module? 🤔

Reply:
mmm but we don't really need to register the tagging module, do we?

Reply:
As we already decided, meanwhile this component is generic, no sense to register any x-module inside